src/dotnet6/http-refit/AzFuncUni.Http/Program.cs

```diff
@@ -26,13 +27,19 @@ public static void Main()
 		private static void ConfigureServices(HostBuilderContext builder, IServiceCollection services)
 		{
 			services
-				.AddHttpClient("HttpBinOrgApi", (provider, client) =>
-				{
-					client.BaseAddress = new System.Uri(HttpBinOrgApiHost);
-					client.DefaultRequestHeaders.Add("Accept", "application/json");
-					client.DefaultRequestHeaders.Add("User-Agent", "dotnet-core/3.1");
-				})
+				.AddHttpClient(nameof(IHttpBinOrgApi), ConfigureHttpClient)
+				.AddTypedClient(c => RestService.For<IHttpBinOrgApi>(c));
+
+			services
+				.AddHttpClient(nameof(IHttpBinOrgApi3), ConfigureHttpClient)
 				.AddTypedClient(c => RestService.For<IHttpBinOrgApi3>(c));
 		}
+
+		private static void ConfigureHttpClient(IServiceProvider provider, HttpClient client)
+		{
+			client.BaseAddress = new System.Uri(HttpBinOrgApiHost);
+			client.DefaultRequestHeaders.Add("Accept", "application/json");
+			client.DefaultRequestHeaders.Add("User-Agent", "dotnet-core/3.1");

```

Comment:
Is this an optional header? If so, let's get rid of it. It now refers to NET Core anyway which is also a bit confusing ;).

Reply:
Sure we can remove it.
I was being a little bit fancy 😂.